repldex/discordbot/bot.py

```diff
@@ -7,6 +7,7 @@
 import io
 intents = discord.Intents.default()
 intents.members = True
+client = discord.Client(intents=intents, member_cache_flags=discord.MemberCacheFlags.none())
```

Comment:
Can't you just remove `intents.members = True` or does that not work?

Reply:
i think we need that for fetching members